draft-ietf-quic-qlog-quic-events.md

```diff
@@ -494,10 +494,10 @@ The `version_information` event supports QUIC version negotiation; see {{Section
 6 of QUIC-TRANSPORT}}. It has Core importance level; see {{Section 9.2 of
 QLOG-MAIN}}.
 
-QUIC endpoints each have their own list of of QUIC versions they support. The
+QUIC endpoints each have their own list of QUIC versions they support. The
 client uses the most likely version in their first initial. If the server does
-support that version, it replies with a version_negotiation packet, containing
-supported versions. From this, the client selects a version. The
+not support that version, it replies with a version_negotiation packet,
+containing supported versions. From this, the client selects a version. The
```

Comment:
Not introduced in this PR, but taking the opportunity to fix another nit: The packet is called Version Negotiation packet, according to RFC 8999.

```suggestion
not support that version, it replies with a Version Negotiation packet, which
contains its supported versions. From this, the client selects a version. The
```

Reply:
Thanks. Today I learned that RFC 9000 is not the first QUIC related RFC.